hooks/pre-command

```diff
@@ -30,6 +30,27 @@ plugin_read_list_into_result() {
   [[ ${#result[@]} -gt 0 ]] || return 1
 }
 
+function plugin_read_list() {
+  prefix_read_list "BUILDKITE_PLUGIN_DOCKER_ECR_PUBLISH_$1"
+}
+
+function prefix_read_list() {
+  local prefix="$1"
+  local parameter="${prefix}_0"
+
+  if [[ -n "${!parameter:-}" ]]; then
+    local i=0
+    local parameter="${prefix}_${i}"
+    while [[ -n "${!parameter:-}" ]]; do
+      echo "${!parameter}"
```

Comment:
adding the echo to the existing `plugin_read_list_into_result` i feel would produce the same outcome using the existing function